modules/installation-configuration-parameters.adoc

```diff
@@ -2763,6 +2771,14 @@ For more information, see "VMware vSphere region and zone enablement".
 |String
 endif::agent[]
 
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        server:
```

Comment:
@dfitzmau `server` field  is under `failureDomains` rather than `topology`, pls correct if I'm wrong, other lgtm, thanks.

```
# ./openshift-install explain installconfig.platform.vsphere.failureDomains
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <[]object>
  FailureDomains holds the VSpherePlatformFailureDomainSpec which contains the definition of region, zone and the vCenter topology. If this is omitted failure domains (regions and zones) will not be used.

FIELDS:
    name <string> -required-
      name defines the name of the FailureDomain This name is arbitrary but will be used in VSpherePlatformDeploymentZone for association.

    region <string> -required-
      region defines a FailureDomainCoordinate which includes the name of the vCenter tag, the failure domain type and the name of the vCenter tag category.

    server <string> -required-
      server is the fully-qualified domain name or the IP address of the vCenter server.

    topology <object> -required-
      Topology describes a given failure domain using vSphere constructs

    zone <string> -required-
      zone defines a VSpherePlatformFailureDomain which includes the name of the vCenter tag, the failure domain type and the name of the vCenter tag category.
```

```
# ./openshift-install explain installconfig.platform.vsphere.failureDomains.topology
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <object>
  Topology describes a given failure domain using vSphere constructs

FIELDS:
    computeCluster <string> -required-
      computeCluster as the failure domain This is required to be a path

    datacenter <string> -required-
      datacenter is the vCenter datacenter in which virtual machines will be located and defined as the failure domain.

    datastore <string> -required-
      datastore is the name or inventory path of the datastore in which the virtual machine is created/located.

    folder <string>
      folder is the inventory path of the folder in which the virtual machine is created/located.

    networks <[]string>
      networks is the list of networks within this failure domain

    resourcePool <string>
      resourcePool is the absolute path of the resource pool where virtual machines will be created. The absolute path is of the form /<datacenter>/host/<cluster>/Resources/<resourcepool>.

    template <string>
      template is the inventory path of the virtual machine or template that will be used for cloning.
```

Reply:
Thanks, @sgaoshang . Very useful to know how to check these parameters structures going forward. I have the `openshift-install` binary stored in my local `bin` directory.